designer/test/component-type-edit.test.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react'
-import { render } from 'enzyme'
+import { render, mount } from 'enzyme'
```

Comment:
sorry I didn't catch these before (gh likes to hide big file changes). I assume you're using `render()` for _easier_ access to attributes? I think render is ok if you are just checking the first render (which is fine since you're only testing initial states), but isn't that helpful if you're testing for state changes

Reply:
Agreed! It was already here for proving the rendering of the component screens. I'm using mount to test the behaviour of said inputs